.github/workflows/_meta.yml

```diff
@@ -24,6 +24,11 @@ on:
 
 permissions: {}
 
+defaults:
+  run:
+    # See https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#defaultsrunshell
+    shell: bash
```

Comment:
Should we use shell: `bash -euo pipefail {0}` then?

Reply:
If you click the link, you'll see that `bash` is a shorthand for `bash --noprofile --norc -eo pipefail {0}`.

If we want `-u` as well, I can adjust it accordingly, but I'd go to `bash --noprofile --norc -euo pipefail {0}`

Reply:
Nice, TIL!

In other workflows, we usually use `shell: bash -euo pipefail {0}` (or an option with `-x`); I think it would be nice to keep it consistent at least within the repo.
`-u` saved me a couple of times, so I'd prefer to have it.

We can adjust it with `--noprofile --norc` in the following PRs.

Reply:
ack, sounds good. Pushed.